rm-deployment-status-enhancer-summary/scripts/main.js

```diff
@@ -0,0 +1,78 @@
+
+VSS.ready(function() {
+	var c = VSS.getConfiguration();
+	c.onReleaseChanged(function(release) {
```

Comment:
It seems there is an issue using this fn wherein one cannot call into VSTS/RM service from the extension. I will get more info on this and if possible will use the fn. We can keep this change for the next iteration.